docs/book/src/topics/powervs/creating-a-cluster.md

```diff
@@ -176,37 +158,10 @@ following the steps below.
     --flavor=powervs-create-infra | kubectl apply -f -
   ```
 
-### Deploy a PowerVS cluster with external cloud provider
```

Comment:
do you think we need to keep a section here with mentioning for default template or the point in L39 will suffice?

Reply:
point at L39 should be enough as per me, no need to explicitly mention it again to avoid repetition.